src/nvim/testdir/test_digraph.vim

```diff
@@ -243,6 +243,7 @@ func! Test_digraphs_option()
   call Put_Dig_BS("P","=")
   call assert_equal(['Р']+repeat(["₽"],2)+['П'], getline(line('.')-3,line('.')))
   " Not a digraph: this is different from <c-k>!
+  exe "norm! 3o"
```

Comment:
Could you instead add the Vim-default value for `'backspace'` to [src/nvim/testdir/runtest.vim](https://github.com/neovim/neovim/blob/695703ca97e9330e3dbbeb3153b57b4ca565d045/src/nvim/testdir/runtest.vim#L65-L67) so other tests don't have to account for this?